src/components/Dropdown/DropdownMenu/DropdownMenu.tsx

```diff
@@ -0,0 +1,44 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { Size } from "@utilities/enum";
+import { ReactElement } from "react";
+import DropdownMenuItem, { MenuItem } from "../DropdownMenuItem/DropdownMenuItem";
+import css from "./DropdownMenu.module.css";
+
+export interface DropdownMenuProps {
+    items: MenuItem[][];
+    onChange: (id: string) => void;
+    size?: Size.Small | Size.Large;
+    activeItemId?: string;
+}
+
+export default function DropdownMenu({
+    items,
+    onChange,
+    size = Size.Small,
+    activeItemId = "",
+}: DropdownMenuProps): ReactElement<DropdownMenuProps> {
+    return (
+        <ul className={`${css.menu} ${size === Size.Large ? css.large : ""}`}>
+            {items.map((menuItems, i) => (
+                <li key={i} className={css.divider} data-test-id="dropdown-divider">
```

Comment:
We may want to have an `id` instead of using the index to cache the rendered elements better.

```
Array of MenuItem:
0. [...]
1. [...] -> deleted
2. [...] -> get rerendered even without changes
```

Reply:
i told @NadiaPosch to do it like this for the moment, but you're right and we're with you on this, it's better to change the items from `MenuItem[][]` to something like `{id: string; items: MenutItem[] }[]` this it would also allow to potentially add a title per group or other props

Reply:
Done 👌🏼